bloom-housing/bloom
# feat: add accessibility building features to listing

# Pull Request Template

## Issue Overview

This PR addresses #2727


## Description

Pulls over the accessibility features field from Detroit.

## How Can This Be Tested/Reviewed?

Add and remove the new building features from a listing. In partners, ensure it displays appropriately on the listing form details page and that the current values populate on the edit form. In public, ensure they display in the features section.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
